docs/latest/design/ratelimit.md

```diff
@@ -0,0 +1,66 @@
+# Ratelimiting Design
+
+## Overview
+
+Ratelimiting is a feature that allows the user to limit the number of incoming requests
+to a predefined value based on attributes within the traffic flow.
+
+Here are some reasons why a user may want to implements Ratelimits
+
+* To prevent malicious activity such as DDoS attacks.
+* To prevent applications and its resources (such as a database) from getting overloaded.
+* To create API limits based on user entitlements.
+
+## API
+
+* Here is an example of a ratelimit implemented by the platform engineer that limits requests made
+by every unique client remote address to 100 requests per second, to help mitigate DDoS attacks.
+```
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: RateLimiting 
+metadata:
+  name: ratelimit-per-client-ip
+spec:
+  type: Global
+  rules:
+  - matches:
+    - remoteAddress: {}
+    limit:
```

Comment:
can also be called `rate`, stuck for `limit` for now

Reply:
+1 for `limit`